tests/functional/modules/test_zos_copy_func.py

```diff
@@ -2059,13 +2059,34 @@ def test_copy_pds_loadlib_member_to_pds_loadlib_member(ansible_zos_module,):
         copy_res = hosts.all.zos_copy(
             src="{0}({1})".format(src, member), 
             dest="{0}({1})".format(dest, "MEM1"), 
-            remote_src=True)
+            remote_src=True,
+            is_executable=True)
 
         verify_copy = hosts.all.shell(
             cmd="mls {0}".format(dest),
             executable=SHELL_EXECUTABLE
         )
 
+        copy_uss_res = hosts.all.zos_copy(
```

Comment:
Why are two cases tested in the same test?

Reply:
For the DS to USS all the rest of the code before the copies will be the same.